.github/workflows/buildbook.yaml

```diff
@@ -82,10 +82,6 @@ jobs:
         restore-keys: |
           notebooks-${{ runner.os }}-${{ hashFiles('environment.yml') }}-${{ github.run_id }}-
           notebooks-${{ runner.os }}-${{ hashFiles('environment.yml') }}-
-    - name: Invalidate EarthCARE cache
-      if: github.event_name == 'schedule'
```

Comment:
so we still have a use-case for scheduled builds without this? If not, we might want to remove the schedule trigger as well...

Reply:
The use-case is not there, yet. Therefore I have disable the schedule for now.

I left the safe-guards (e.g. only run nightly builds on the main repo) as they should not harm our current workflows.